src/test/java/tracko/logic/commands/DeleteOrderCommandTest.java

```diff
@@ -78,6 +78,22 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
         assertCommandFailure(deleteOrderCommand, model, Messages.MESSAGE_INVALID_ORDER_DISPLAYED_INDEX);
     }
 
+    @Test
+    public void execute_validIndexSortedList_success() {
+        showOrderAtIndex(model, INDEX_FIRST);
+
+        Order orderToDelete = model.getSortedOrderList().get(INDEX_FIRST.getZeroBased());
+        DeleteOrderCommand deleteOrderCommand = new DeleteOrderCommand(INDEX_FIRST);
+
+        String expectedMessage = String.format(DeleteOrderCommand.MESSAGE_DELETE_ORDER_SUCCESS, orderToDelete);
+
+        Model expectedModel = new ModelManager(model.getTrackO(), new UserPrefs());
+        expectedModel.deleteOrder(orderToDelete);
+        showNoOrder(expectedModel);
```

Comment:
I'm just curious, what does `showNoOrder(Model model)` do? I don't really understand the method :( 

There's nothing wrong with the code hahahaha I'm just trying to grapple with what the function does.

Reply:
does that mean that `expectedModel` does not contain any orders in its OrderList?

Reply:
Nope, it just shows no orders in the UI. It's like using `findo` without a matching string so no orders show up. 

https://docs.oracle.com/javase/8/javafx/api/javafx/collections/transformation/FilteredList.html#setPredicate-java.util.function.Predicate-

Reply:
ah I see, thank you :)